blobmodel/model.py

```diff
@@ -215,7 +211,7 @@ def animate(i):
             ani.save(gif_name, writer='ffmpeg', fps=fps)
         plt.show()
 
-    def integrate(self, file_name='2d_blobs.nc', speed_up=False, truncation_Lx=3):
```

Comment:
I don't see the point in this either, why would you call integrate if you don't want to save the data?